src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -2,10 +2,13 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns:context="http://www.springframework.org/schema/context"
        xsi:schemaLocation="http://www.springframework.org/schema/beans
-  http://www.springframework.org/schema/beans/spring-beans.xsd
-  http://www.springframework.org/schema/context
-  http://www.springframework.org/schema/context/spring-context.xsd"
->
-    <bean id="greetingService" class="org.noteskeeper.server.GreetingService"/>
+           http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
```

Comment:
XML-based Spring configs are used in official examples to define beans.
https://jersey.java.net/documentation/latest/spring.html + https://github.com/jersey/jersey/tree/2.25.1/examples/helloworld-spring-webapp